docs/alternator/compatibility.md

```diff
@@ -43,7 +43,7 @@ need to use the slower LWT for all writes - to correctly isolate concurrent
 writes. However, if Alternator is told that a certain workload does not have
 any read-modify-write operations, it can do all writes with the faster
 non-LWT write. Furthermore, if Alternator is told that a certain workload
-does have do both write-only and read-modify-write, but to *different* items,
+does have to do both write-only and read-modify-write, but also to do *different* items,
```

Comment:
This is the wrong fix for this mistake. The word "also" in your suggestion is not relevant. The proper correction would be to remove the superfluous word "do" (or alternatively, the "have"). I would also add the word "operations" because write-only is more appropriately an adjective, not a noun. The fixed version would be:

> Furthermore, if Alternator is told that a certain workload does have ~~do~~ both write-only and read-modify-write **operations**, but to different items, it could use LWT only for the read-modify-write operations.

Reply:
By the way, please make the commit title more self-contained for people who read the mailing list or "git log". "update" doesn't say why (was it out of date?) and "compatibility.md" is not something that most people understand what it refers to.
A better commit title can be: **docs: copy-edit alternator/compatibility.md**.

Reply:
Ok, thanks for the helpful comments.